### PR TITLE
init database and oauth_token

### PR DESCRIPTION
**Pull request by tonkaew131**
Create database and oauth_token when bot's running first time

### CODE REVIEW

**Comment by tonkaew131** [comment]
add !rich and change getPlayers() to be more readable

**Comment by thananon** [comment]
Please rebase on top of current main.

**Comment by tonkaew131** [comment]
Rebased and resolved conflicts

File: 9armbot/9armbot.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 require('dotenv').config({ path: './../.env'})
 const tmi = require('tmi.js');
 const fs = require('fs');
-var oauth_token = fs.readFileSync('oauth_token', 'utf8');
+const XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
```
Comment (Aitthi) [inline 9armbot/9armbot.js:4]: No package is used. `XMLHttpRequest` now, please delete it.

**Comment by thananon** [comment]
No update. I will just close this.

**Pull request closed without merging**